syndeo-template/pom.xml

```
@@ -78,6 +78,11 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+            <dependency>
+                <groupId>org.apache.beam</groupId>
+                <artifactId>beam-sdks-java-io-kafka</artifactId>
+                <version>2.47.0-SNAPSHOT</version>
```

Should we use `${beam.snapshot.version}`?
```suggestion
                <version>${beam.snapshot.version}</version>
```

---

ah yes correct!